real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/nodes/RBSMNode.java

```diff
@@ -97,28 +103,33 @@ public boolean matchDataSample(byte[] sample) {
 	}
 
 	@Override
-	public int matchDataMinSize() {
+	public int matchDataMinSize() 
+	{
 		return 2*RBSerial.MSG_LEN;
 	}
 
 	@Override
 	//must be the same as the baud rate of the arduino 
-	public int baudRate() {
+	public int baudRate() 
+	{
 		return 115200;
 	}
 
 	@Override
-	public int peel(byte[] buffer, int start, int bytes_available) {
+	public int peel(byte[] buffer, int start, int bytes_available) 
+	{
 		// The Encoder sends 3 types of messages
 		//  - Encoder ticks since last message (keep)
 		//  - Number of ticks since last reset
 		//  - Timestamp since reset
 		RBPair rbp = RBSerial.peel(buffer, start, bytes_available);
-		switch(rbp.getNumberOfBytesRead()) {
+		switch(rbp.getNumberOfBytesRead()) 
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [A switch statement does not contain a break](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/1274095682/issues/source?bid=2938665&fileBranchId=2938741#l126)